packages/babel-helpers/src/helpers.js

```diff
@@ -430,10 +430,18 @@ helpers.objectDestructuringEmpty = template(`
 helpers.objectWithoutProperties = template(`
   (function (obj, keys) {
     var target = {};
-    for (var i in obj) {
-      if (keys.indexOf(i) >= 0) continue;
-      if (!Object.prototype.hasOwnProperty.call(obj, i)) continue;
-      target[i] = obj[i];
+    if (obj == null) return target;
+    var i, key, toExclude = [];
+    for (i in keys) {
+      key = keys[i];
+      toExclude.push(typeof key == "symbol" ? key : "" + key)
```

Comment:
Should I use a `.map` instead?

This also probably should be a `Set` instead of tests with `indexOf`